src/less/components/file/index.less

```diff
@@ -60,4 +60,4 @@ hx-file-tile {
   &[invalid] {
     border-color: @red-900;
   }
-}
+}
```

Comment:
Try adding a new line here to remove any changes from this file.

You should have a total of **6** `Files changed` for this PR.

Reply:
This shouldn't create any issues.  Merging.